### DISCUSSION TITLE
Cross Compilation for arm Architecture or Build from Source in arm

### DISCUSSION THREAD

**Discussion by harshakhmk** [qdrant/qdrant#3250] [General]
I am exploring Qdrant and tried out few sample examples, Does Qdrant support cross compilation or build from source for arm Architecture 

If yes, can I get the complete guide of steps to run, build and start the server on arm Architectures

Any leads on this, would be highly appreciated

**Comment by generall** [comment]
Hi @harshakhmk, yes qdrant can be built for Arm. Moreover, our default docker image is already compiled for both x86 and arm platforms.

The exact instructions you can find in Dockerfile

**Comment by harshakhmk** [comment]
> Hi @harshakhmk, yes qdrant can be built for Arm. Moreover, our default docker image is already compiled for both x86 and arm platforms.
> 
> The exact instructions you can find in Dockerfile

Ok, is cross compilation also possible for arm in qdrant

**Comment by generall** [comment]
yes, we build arm from default github workers
